scss/_patterns_equal-height-row.scss

```diff
@@ -18,18 +20,28 @@
       margin-top: -1px;
       position: relative;
 
+      // If first row item contains an image container, apply top margin to
+      // match top margin from deprecated media container component
+      .p-equal-height-row__item:first-child .p-image-container {
```

Comment:
I don't think targeting class names from another component is ideal. Could we achieve this some other way? Maybe a modifier class?

Reply:
I think this relates to another issue that we have.

In the initial design for horizontal rules (`p-rule` component and its vatiants) rules did not have a bottom margin. It was OK when they have been followed by text content, but images required a top margin (and so the now deprecated `p-media-container` was born to add this margin).

But after struggling it for some time Lyubo decided that it's OK for the text to be further away from the horizontal lines, so that horizontal lines can have bottom margin, and images don't need to have the top margin anymore.

I guess the issue is, that currently only class-less base `<hr>` has bottom margin, but `p-rule` doesn't.

There actually is a PR by Pete to address that: https://github.com/canonical/vanilla-framework/pull/5241 but it kind of got stuck in review limbo.

So, to summarise. The equal heights should not be doing anything to add any margins to images. If there are lines added by the pattern, the lines themselves should have a bottom margin to move everything below them by 0.5rem.
If the lines are `p-rule` component, it's the rule component that needs to be updated to have bottom margin.

I guess we can achieve that by finishing the Pete's PR.